### ISSUE TITLE
Mount prevent second run

### ISSUE THREAD

**Issue by Ratio2** [Zygo/bees#267]
1.Run "sudo /usr/local/sbin/beesd UUID"
2. Close it with SIGINT(ctrl+c)/SIGTERM
3. Run "sudo /usr/local/sbin/beesd UUID"

mount: /run/bees/mnt/UUID: /dev/... already mounted on /.

But if we use "sudo systemctl stop bees@UUID" then mount automatically closed and no problem occurred.

Very strange. I not found special handling in source code. Systemd behavior?

**Comment by lilydjwg** [comment]
> Systemd behavior?

Yes. The systemd service uses a private mount which gets cleaned up after service is stopped. When you manually run beesd, you'll need to unmount that mountpoint yourself (that's why I don't do that).
